pkg/utils/util.go

```diff
@@ -788,7 +788,7 @@ func FormatNewDisk(readOnly bool, source, fstype, target string, mkfsOptions, mo
 	_, err := diskMounter.Exec.Command("mkfs."+fstype, args...).CombinedOutput()
 	if err == nil {
 		// the disk has been formatted successfully try to mount it again.
-		klog.Infof("Disk format succeeded, pvName: %s elapsedTime: %+v ms", pvName, time.Now().Sub(startT).Milliseconds())
+		klog.Infof("Disk format succeeded, pvName: %s elapsedTime: %+v ms", pvName, time.Since(startT).Microseconds())
```

Comment:
I suppose this change is not intended? because the unit in the message is not changed.

```suggestion
		klog.Infof("Disk format succeeded, pvName: %s elapsedTime: %+v ms", pvName, time.Since(startT).Milliseconds())
```